app/questionnaire/questionnaire_manager.py

```diff
@@ -144,13 +144,14 @@ def update_state(self, item_id, user_input):
     def _append(self, node):
         if not self._first:
             self._first = node
+            self._current = node
```

Comment:
this appears to be a reversion of the change made to move this code outside the if/else statement from before.